Eniko/index.js

```diff
@@ -0,0 +1,151 @@
+// 1. Person Class
+// Create a class called Person which accepts the name of a person as a string, and his/her age as a number. The Person class should have a method called describe which returns a string with the following syntax: "name, age years old". So, for example, if John is 19 years old, then the function describe of his object will return "John, 19 years old".
+class Person {
+    constructor(name, age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    render() {
+        return `${this.name}, ${this.age} years old`
+    }
+}
+
+const person = new Person("Annie", "111");
+console.log(person.render());
+
+// 2. Volume
+// Write a JavaScript program to get the volume of a cylinder with four decimal places using a class.
+class Cylinder {
+    constructor(radius, height) {
+        this.radius = radius;
+        this.height = height;
+    }
+
+    render() {
+        let volume = 3.14 * (this.radius * this.radius) * this.height;
+        return `A cylinder with ${this.height} cm height and ${this.radius} cm radius has the volume of ${volume} cm3` 
+    }
+}
+const cylinder = new Cylinder(5, 10);
+console.log(cylinder.render());
+
+
+// Note: Volume of a cylinder : V = πr^2h - r is the radius and h is the height of the cylinder.
+
+// 3. Tick Tock
+// Rewrite the following code to use the "class" syntax.
+
+class Clock {
+    constructor({template}) {
+        this.template = template;
```

Comment:
Task 3 ==> Please also add this.timer; to constructor after this.template = template